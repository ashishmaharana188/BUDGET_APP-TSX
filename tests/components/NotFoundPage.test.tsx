import React from "react";
import { test, expect } from "vitest";
import { render } from "@testing-library/react";
import { MemoryRouter } from "react-router-dom";

import NotFoundPage from "../../src/components/NotFoundPage";

test("should render notFoundPage", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <NotFoundPage />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
