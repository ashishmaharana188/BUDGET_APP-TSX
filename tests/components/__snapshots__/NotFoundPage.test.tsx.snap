// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should render notFoundPage 1`] = `
<DocumentFragment>
  <div>
    404! 
    <a
      data-discover="true"
      href="/"
    >
       Go Home
    </a>
  </div>
</DocumentFragment>
`;
